FireboltNETSDK/Client/FireboltConnection.cs

```diff
@@ -189,14 +191,22 @@ public override async Task<bool> OpenAsync(CancellationToken cancellationToken)
             }
         }
 
-        public GetEngineUrlByEngineNameResponse SetEngine(string? engineName)
+        public GetEngineUrlByEngineNameResponse SetEngine(string engineName)
         {
+            if (Client is null)
```

Comment:
Can we make this consistent with line 172? Either with or without `{}`

Reply:
done